ParquetUnitTests/Sandbox/Parquets/BlockUnitTest.cs

```diff
@@ -8,21 +8,19 @@ public class BlockUnitTest
         [Fact]
         public void ToughnessCannotBeSetBelowZeroTest()
```

Comment:
I think this and the below test's names should be swapped.

Reply:
Got my mins and maxs backwards — thanks!
Fixed in c6df0fc.